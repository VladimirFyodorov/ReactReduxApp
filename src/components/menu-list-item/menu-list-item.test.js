import React from 'react';
import MenuListItem from './index';
import {shallow} from 'enzyme';

// npm run test
describe('Testing <MenuListItem/>', () => {
  const itemData = {title: '', price: '', url: '', category: ''};
  const fakeFunc = () => console.log('fake');
  const item = shallow(<MenuListItem menuItem={itemData} onAddToCart={fakeFunc}/>);

  describe('Testing snap & state', () => {
    it('MenuListItem has rendered correctly', () => {
      expect(item).toMatchSnapshot();
    })
    // it('MenuListItem props "menuItem" is object', () => {
    //   expect({}).toBeObject();
    // })
  })

  // describe('Testing methods of the object:', () => {
  //   it('method_1', () => {
  //     item.instance().method_1();
  //     // method_1 shange 'error' in 'state' to false
  //     expect(item.state().error).toBeFalsy();
  //   })
  // })
})



// import renderer from 'react-test-renderer';
// describe('Testing <MenuListItem/>', () => {
//   it('MenuListItem has rebdered correctly', () => {
//     // const itemData
//     const item = renderer.create(<MenuListItem/>).toJSON();
//     expect(item).toMatchSnapshot();
//   })
// })