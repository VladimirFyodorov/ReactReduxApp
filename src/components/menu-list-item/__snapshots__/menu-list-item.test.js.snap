// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <MenuListItem/> Testing snap & state MenuListItem has rendered correctly 1`] = `ShallowWrapper {}`;